_src-lp/blocks/b-productswithselectors/__TESTS__/b-productswithselectors.spec.js

```diff
@@ -0,0 +1,67 @@
+// eslint-disable-next-line import/extensions
+import mockData from './mock-data.json';
+// mock everything that is not related to the actual main scope
+jest.mock('../../../scripts/lib-franklin.js', () => ({
+  ...jest.requireActual('../../../scripts/lib-franklin.js'),
+  loadHeader: jest.fn(),
+  loadFooter: jest.fn(),
+}));
+
+describe('b-productswithselectors block', () => {
+
+  beforeEach(() => {
+    jest.resetModules();
+    jest.clearAllMocks();
+  });
+
+  describe('b-productswithselectors', () => {
+
+    // init / bootstrap page()
+    document.body.innerHTML = mockData.b_productswithselectors;
+    import('../../../scripts/scripts.js');
+
+    let b_productswithselectors;
+
+    beforeEach(() => {
+      b_productswithselectors = document.querySelector('.b-productswithselectors-container');
+    });
+
+    test('check if products have been added to the window.productsListCount', () => {
+        expect(window.productsListCount).toBeTruthy();
+    });
+
+    test('check if the number of servers and mailboxes changes based on number of devices selected', () => {
+      const UsersSelectorBox = b_productswithselectors.querySelectorAll('.selectorBox')[0];
```

Comment:
variable name should be lowercase camelcase: usersSelectorBox, capital letter is used for class names

Reply:
updated variable name